src/Mutation/PostObjectUpdate.php

```diff
@@ -160,8 +160,13 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 			/**
 			 * Throw an exception if the post failed to update
 			 */
-			if ( 0 === $post_id || is_wp_error( $updated_post_id ) ) {
```

Comment:
`0 === $post_id` is already ruled out on [line 80](https://github.com/wp-graphql/wp-graphql/blob/d5a2c6c1b77e2aff401f971ad3e9fe0863f850d1/src/Mutation/PostObjectUpdate.php#L80), so we can use the same logic /throws we do elsewhere.